RecoParticleFlow/PFProducer/python/particleFlow_cfi.py

```diff
@@ -70,12 +70,24 @@
     maxEeleOverPout = cms.double(0.2),
     maxDPhiIN = cms.double(0.1)    
     ),
+    electron_protectionsForBadHcal = cms.PSet(
+        enableProtections = cms.bool(False),   
+        full5x5_sigmaIetaIeta = cms.vdouble(0.0106, 0.0387), # EB, EE; 94Xv2 cut-based medium id
+        eInvPInv = cms.vdouble(0.184, 0.0721),
```

Comment:
this value was ```cms.vdouble(0.209, 0.132)``` before.
Now it's changed to be exactly the same as dPhi. This looks suspicious as if one of the two is a typo

Reply:
indeed, the dPhi is a typo. fixing